tests/subset_ROI_test.py

```diff
@@ -0,0 +1,158 @@
+# %%
+from pathlib import Path
+
+import geopandas as gpd
+import matplotlib
+import numpy as np
+from scipy.spatial.transform import Rotation
+from shapely import MultiPolygon, Point, Polygon
+from tqdm import tqdm
+
+from geograypher.cameras.cameras import PhotogrammetryCameraSet
+from geograypher.constants import MATPLOTLIB_PALLETE, VIS_FOLDER
+from geograypher.utils.example_data import (
+    create_non_overlapping_points,
+    create_scene_mesh,
+)
+
+# Where to save content
+CONCEPT_FIGURE_ROOT = Path(VIS_FOLDER, "concept_figure_content")
```

Comment:
This is taken from an example and I'm pretty sure that a lot of this code could be safely removed. We should only need content related to the geometry of the cameras and RoIs.